src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java

```diff
@@ -12,7 +12,7 @@
 
 @WebServlet("/shopping-carts/products/delete")
```

Comment:
for some controllers you are using annotation but for some not?

Reply:
It is so because first I didn't know about @WebServlet annotation and began to use it only after first homework. 
But you are right, it should be one style, so now I use annotations everywhere.